src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -268,7 +258,7 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
 
   updateWindowDimensions = () => {
     this.setState({
-      width: window.innerWidth,
+      isDesktop: window.innerWidth > minDesktopWidth,
```

Comment:
I know the code has been isDesktop in the past but I wonder if it would be better to detect a touch screen instead for stuff like pinch to zoom. We have something like this in Infield for example: https://github.com/cognitedata/infield/blob/7ea6f6849217bd89ca489860013b6773196a0cdf/src/hooks/useHasTouchScreen.ts

Reply:
Agree with you and I think it's quite weird to rely on screen width in a component. 

But it will be a change in behavior I think. I would prefer to leave things as they are for now. But when we bring all this stuff into infield codebase, we'll have more freedom and confidence to change things as we like and we can address it there later.